Source/Core/AudioCommon/Mixer.h

```diff
@@ -75,6 +78,10 @@ class CMixer final
   MixerFifo m_wiimote_speaker_mixer{this, 3000};
   unsigned int m_sampleRate;
 
+  soundtouch::SoundTouch m_sound_touch;
+  double m_stretch_ratio = 1.0;
+  void StretchAudio(short* samples, unsigned int actual_samples, unsigned int num_samples);
```

Comment:
Functions shouldn't be intermixed with variable declarations. The general hierarchy of organization is:

1. classes, structs, enums, unions
2. functions
3. variables